### DISCUSSION TITLE
How to use graphson to import properties with multiple values?

### DISCUSSION THREAD

**Discussion by bidiudiu** [JanusGraph/janusgraph#2759] [General]
This is a very simple point, but I think it may help someone save a little time. 
Please note that **before using graph.io to import data, you need to define the cardinality of a multi-valued property as LIST through mgmt**. Otherwise the schema with cardinality as SINGLE will be automatically created when importing, causing multi-values ​​to be overwritten into single values.

Sample:  `{"id":4320,"label":"feature","properties":{"multi_code": [{"id": 1,"value": "A"},{"id": 2,"value": "B"}]}}`

**Comment by vtslab** [comment]
Good point. Indeed, it would not hurt to have a section in the ref docs pointing at the availability of TinkerPop graph.io / g.io() with such janusgraph-specific details!
